### DISCUSSION TITLE
Django Async, best way?

### DISCUSSION THREAD

**Discussion by jfmatth** [piku/piku#332] [Q&A] [answered]
I was thinking of using Django Async views and see that UWSGI doesn't support them directly.

In that case, would it make sense to just use the WEB worker and run uvicorn directly, as referenced here by Django
```https://docs.djangoproject.com/en/5.0/howto/deployment/asgi/uvicorn/#running-django-in-uvicorn```

If i do this, I still get the ability to use UWSGI for other workers, etc. right?

**Comment by chr15m** [comment, marked as answer]
@jfmatth yep that's exactly right. You can use a mix of both for different apps.

**Reply by jfmatth** [reply]
Thanks guys, Happy Holidays to you and your families.

**Reply by chr15m** [reply]
You too! :tada:

**Reply by seguri** [reply]
What does this mean in practice, considering the attached link? Doing something like this in the Procfile?

```
web: python -m uvicorn myproject.asgi:application
```
